src/status_im2/common/floating_button_page/floating_container/style.cljs

```diff
@@ -0,0 +1,18 @@
+(ns status-im2.common.floating-button-page.floating-container.style
+  (:require [react-native.platform :as platform]))
+
+(def container
+  {:width      "100%"
+   :margin-top :auto
```

Comment:
this is needed to push the button to the bottom of screen to sit on top of the keyboard view